index.html

```diff
@@ -59,25 +59,55 @@
     </div>
   </nav>
   <section id="about-us">
-    <div class="card">
-      <div class="row g-0">
-        <div class="col-md-4">
-          <img src="\img\waw.png" class="img-fluid rounded-start" alt="Group's photo">
+    <div class="container">
+      <div class="row align-items-center justify-content-between">
+        <div class="col-12 col-lg-6 mb-6 mb-lg-0">
+          <img class="img-fluid rounded-3" src="/img/waw.png" alt="Group's photo">
         </div>
-        <div class="col-md-8">
-          <div class="card-body">
-            <h6 class="card-title">About Us</h6>
-            <h1 class="card-title" style="text-align: center;">The most trusted job market</h1>
-            <p class="card-text">
-            <ul class="list-group list-group-flush" style="text-align: center;">
-              <li class="list-group-item">Free of charge</li>
-              <li class="list-group-item">Safe and legal</li>
-              <li class="list-group-item">Interview planning included</li>
-            </ul>
-            </p>
-            <div class="d-grid gap-2 d-md-flex justify-content-md-center">
-              <button type="button" class="btn btn-secondary btn-lg">Join Us!</button>
+        <div class="col-12 col-lg-6 col-xl-5">
+          <div> <span class="badge badge-primary-soft p-2">
+              <i class="la la-exclamation ic-3x rotation"></i>
+            </span>
```

Comment:
You should remove the `span` element, since it is not used.

```suggestion
          <div>
```